app/src/sharedTest/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -239,66 +188,28 @@ class TopicPracticeFragmentTest {
   }
 
   @Test
-  fun testTopicPracticeFragment_loadFragment_selectSubtopics_deselectSubtopics_isSuccessful() {
+  fun testTopicPracticeFragment_loadFragment_selectSubtopics_thenDeselect_selectsCorrectTopic() {
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(
-        allOf(
-          withText(TopicTab.getTabForPosition(2).name),
-          isDescendantOfA(withId(R.id.topic_tabs_container))
-        )
-      ).perform(click())
-      testCoroutineDispatchers.runCurrent()
+      clickPracticeTab()
+      clickPracticeItem(position = 1, targetViewId = R.id.subtopic_check_box)
+      clickPracticeItem(position = 1, targetViewId = R.id.subtopic_check_box)
       onView(
         atPositionOnView(
           R.id.topic_practice_skill_list,
           1,
           R.id.subtopic_check_box
         )
-      ).perform(click())
-      onView(
-        atPositionOnView(
-          R.id.topic_practice_skill_list,
-          1, R.id.subtopic_check_box
-        )
-      ).perform(click())
-      onView(
-        atPositionOnView(
-          R.id.topic_practice_skill_list,
-          1, R.id.subtopic_check_box
-        )
       ).check(matches(not(isChecked())))
     }
   }
 
   @Test
-  fun testTopicPracticeFragment_loadFragment_selectSubtopics_deselectsubtopics_startButtonIsInactive() { // ktlint-disable max-line-length
+  fun testTopicPracticeFragment_loadFragment_selectSubtopics_deselectSubtopics_startButtonIsInactive() { // ktlint-disable max-line-length
```

Comment:
Can we remove the max-line disable directive now?

Reply:
done